### ISSUE TITLE
`pyperclip.__version__` cannot be recognized

### ISSUE THREAD

**Issue by op200** [microsoft/pyright#11687]
**Describe the bug**
[pyperclip](https://github.com/asweigart/pyperclip) is a common library (`pip install pyperclip`)  
Its `__version__` cannot be recognized

**Code or Screenshots**
```python
import pyperclip
print(pyperclip.__version__)
```

```pwsh
PS C:\Users\Administrator\Documents\Python\test> pyright --version
pyright 1.1.413
PS C:\Users\Administrator\Documents\Python\test> pyright
c:\Users\Administrator\Documents\Python\test\2.py
  c:\Users\Administrator\Documents\Python\test\2.py:2:17 - error: "__version__" is not a known attribute of module "pyperclip" (reportAttributeAccessIssue)
1 error, 0 warnings, 0 informations
PS C:\Users\Administrator\Documents\Python\test> ty version                                                                                    
ty 0.0.75 (a553dcc1a 2026-08-26)
PS C:\Users\Administrator\Documents\Python\test> ty check  
All checks passed!
PS C:\Users\Administrator\Documents\Python\test> py 2.py                                  
1.11.0
```

Windows 10 22H2
Python 3.14.7

**Comment by op200** [comment]
## Windows Sandbox test

### Installed types
```
C:\Users\WDAGUtilityAccount\Desktop>pip install types-pyperclip
Collecting types-pyperclip
  Downloading types_pyperclip-1.11.0.20260508-py3-none-any.whl.metadata (1.8 kB)
Downloading types_pyperclip-1.11.0.20260508-py3-none-any.whl (7.6 kB)
Installing collected packages: types-pyperclip
Successfully installed types-pyperclip-1.11.0.20260508
```

mypy
```
C:\Users\WDAGUtilityAccount\Desktop>mypy --version
mypy 2.3.1 (compiled: yes)

C:\Users\WDAGUtilityAccount\Desktop>mypy 2.py
2.py:2: error: Module has no attribute "__version__"  [attr-defined]
Found 1 error in 1 file (checked 1 source file)
```

pyright
```
C:\Users\WDAGUtilityAccount\Desktop>pyright --version
pyright 1.1.413

C:\Users\WDAGUtilityAccount\Desktop>pyright 2.py
c:\Users\WDAGUtilityAccount\Desktop\2.py
  c:\Users\WDAGUtilityAccount\Desktop\2.py:2:17 - error: "__version__" is not a known attribute of module "pyperclip" (reportAttributeAccessIssue)
1 error, 0 warnings, 0 informations
```

ty
```
C:\Users\WDAGUtilityAccount\Desktop>ty version
ty 0.0.75 (a553dcc1a 2026-08-26)

C:\Users\WDAGUtilityAccount\Desktop>ty check
error[unresolved-attribute]: Module `pyperclip` has no member `__version__`
 --> 2.py:2:7
  |
2 | print(pyperclip.__version__)
  |       ^^^^^^^^^^^^^^^^^^^^^

Found 1 diagnostic
```


### Uninstalled types

mypy
```
C:\Users\WDAGUtilityAccount\Desktop>mypy 2.py
2.py:1: error: Library stubs not installed for "pyperclip"  [import-untyped]
2.py:1: note: Hint: "python3 -m pip install types-pyperclip"
2.py:1: note: (or run "mypy --install-types" to install all missing stub packages)
2.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```

pyright
```
C:\Users\WDAGUtilityAccount\Desktop>pyright --version
pyright 1.1.413

C:\Users\WDAGUtilityAccount\Desktop>pyright
c:\Users\WDAGUtilityAccount\Desktop\2.py
  c:\Users\WDAGUtilityAccount\Desktop\2.py:2:17 - error: "__version__" is not a known attribute of module "pyperclip" (reportAttributeAccessIssue)
1 error, 0 warnings, 0 informations
```

ty
```
C:\Users\WDAGUtilityAccount\Desktop>ty version
ty 0.0.75 (a553dcc1a 2026-08-26)

C:\Users\WDAGUtilityAccount\Desktop>ty check
All checks passed!
```

**Comment by bschnurr** [comment]
This diagnostic comes from the `types-pyperclip` stub package. Its stub does not declare `__version__`, so Pyright correctly reports the attribute as unknown when the stub is installed. Mypy and ty report the same error when they use this stub.

Please report this against [python/typeshed](https://github.com/python/typeshed) so that `__version__` can be added to the `pyperclip` stub.
